### ISSUE TITLE
BUG (High Prio): "Convert to CDM" feature has duplicate naming, causing error

### ISSUE THREAD

**Issue by Zwky26** [OHDSI/Perseus#5]
_Originally posted by @Zwky26 in https://github.com/OHDSI/Perseus/issues/4#issuecomment-1242061026_
Error as originally posted here is caused by multiple entities being named "person_id". In a test mapping, I included only a single column and met the minimum requirements for pushing a "Convert to CDM" request. 
![image](https://user-images.githubusercontent.com/59004211/189399607-d5f9ce17-8add-40f0-949a-5dd214d19a45.png)

Within the generated SQL preview, this means that the only mentions of "person_id" are those that are hardcoded within Perseus' logic. This might mean the error is independent of the mapping, and needs to be renamed to avoid ambiguity
![image](https://user-images.githubusercontent.com/59004211/189400299-7ddaccbb-86bf-4a13-be83-c4d1ffc74c12.png)

**Comment by bradanton** [comment]
@Zwky26 Could you please share etl file?

![image](https://user-images.githubusercontent.com/13117785/189610217-60150f9c-80db-42ae-a4a2-903518895bec.png)

**Comment by Zwky26** [comment]
The configuration I used to get that error has proprietary data for some tables. When I removed those tables I get a different error. 

I restructured the mapping to be very basic, with only a single table for now, and am getting new error messages. It first throws an error saying I need to include an "aou" table, even if aou is never referenced. (Err1.etl)
![image](https://user-images.githubusercontent.com/59004211/189687262-c3432f96-99f9-48c2-94d7-d9be99374a95.png)

I then, add an aou table, and get another error saying "patient" table does not exist, even though it is never referenced. 
![image](https://user-images.githubusercontent.com/59004211/189689962-9127c8f3-6b6c-4925-b83a-c70686b3c728.png)

I then add a patient table, and get another peculiar error. It seems that past queries, "Create View" tables, and aliases from a previous instance/mapping are still being called, without being defined or written. I am using a new instance and fresh login but the logic somehow references a column/table that is not specified. I've sent the .etl files via email as github does not allow sharing of that filetype
![image](https://user-images.githubusercontent.com/59004211/189691741-789e4548-1ed2-4fbc-af51-6b6cb5ce7d3b.png)
